docs/DeveloperGuide.adoc

```diff
@@ -453,6 +453,97 @@ does not have to be aware of new `Predicates` +
 
 ---
 
+=== Playing Music
+Unify have a built-in background music player. Users have the choice of playing some music to relax their mind and reduce their stress.
+
+The `music` command is extended with an additional command behind it. For example, `music play`, has play as the extended command, this will `play` command at the back means that the system will play a music track. While `music pause` will pause the current music track.
+
+All these different actions are executed in the same `MusicCommand` class. This keep the commands easier to remember for the user as well, as anything related to `music` will start with `music` as a command.
```

Comment:
This makes the commands easier to remember for the user as well, as anything related to music will start with music as a command.